### PR TITLE
Edit fix

### LINKED ISSUE #164: Redesign edit page

### PR DESCRIPTION
**Pull request by Karol-2**
Not much but it's honest work.

- Added a popup component that shows a message for a few seconds
- Changed user pics to open-source ones
- Fixed width of inputs on edit page
- Removed unused files
- Cleaned up some directories

### CODE REVIEW

**Review by Piterson25 — APPROVED** [review]
Really nice changes!
Popup is amazing :D

Also:
firefox_LLIqrc7vaX
🦎

File: frontend/src/components/Popup.tsx [hunk 1]
```diff
@@ -2,14 +2,14 @@ import { motion } from "framer-motion";
 import { useEffect } from "react";
 
 
-interface ModalInterface {
+interface Popup {
```
Comment (gf-rog) [inline frontend/src/components/Popup.tsx:4]: Here you have both an interface called Popup and a function called Popup.
That means there's a name collision. I would call this `PopupProps`

**Review by gf-rog — APPROVED** [review]
image
Watermelon Musk

File: frontend/src/components/Popup.tsx (hunk 1 above)
Comment (Karol-2) [reply frontend/src/components/Popup.tsx]: done!

**Review by JANKROL123 — APPROVED** [review]
looks nice

**Pull request merged**